examples/hangman.rs

```diff
@@ -0,0 +1,274 @@
+//! A hangman game to guess the hidden word.
+// To run this example, use the command: cargo run --features unstable --example hangman
+#[cfg(all(not(feature = "unstable")))]
+compile_error!("This example relies on unstable features. Run with `--features unstable`");
+
+use std::io::{Error, Write};
+
+use graphic::HangmanDesign;
+use turtle::{rand::RandomSlice, Drawing, Point, Speed, Turtle};
+
+// A small word list to choose from
+const WORDS: &[&str; 28] = &[
```

Comment:
I appreciate the desire to always have a different game every time you run the example, but does having a small word list have much benefit over just hardcoding the word? I think it might actually be better to save these lines and have just a single hardcoded word variable at the top of the program as you previously did. Keep in mind that examples in this crate are more about showing how to make things with the turtle, not about making fully-fledged applications. (I would be very excited to see a fully-fledged version of this outside of this repo! I'm just trying to manage the size and complexity of what we keep here.)

Reply:
With a list of 28 words you can play the game about 40 times... - I think it should be part of the example as without this it is not really a game, someone who just learns can then extend more easily and does not have to research the choose function which is usually quite hard to find for beginners - at least with those I teach... I could move the declaration to the end though.

Reply:
That's fair. It doesn't have to be right at the end, but let's at least move it so it isn't the first thing in the file. (Having it first pushes down the more important code in the file.)

Reply:
It is now right before the `struct HangmanDesign`